### DISCUSSION TITLE
Question about example input file in combined module

### DISCUSSION THREAD

**Discussion by Bala-1005** [idaholab/moose#16909] [Q&A Modules: Solid mechanics] [answered]
Hello everyone,

I am trying to understand the example file given in the directory "/moose/modules/combined/examples/phase_field-mechanics/kks_mechanics_VTS.i". 

I have a question about how the elasticity tensor is constructed. Does the C_ijkl take values and multiply them with 10^9 to convert it into GPa scale or does it take values as it is?

Thanks,
Bala

**Comment by jessecarterMOOSE** [comment, marked as answer]
Looks like the values in the input file have already been converted from GPa. The unit conversion is in the referenced paper. Right @laagesen ?

**Reply by laagesen** [reply]
Yes that's right, thanks @jessecarterMOOSE - they are non-dimensionalized by the factor by 2 x 10^9 J/m^3 = 2 GPa (see p.12 of the paper) . So if you multiply the elastic constants in the input file by 2 GPa you will get the numbers in Table 1.

**Reply by Bala-1005** [reply]
Thank you @laagesen and @jessecarterMOOSE.
